scripts/exrrfs_fcst.sh

```diff
@@ -28,14 +28,18 @@ else
   ${cpreq} ${COMINrrfs}/${RUN}.${PDY}/${cyc}${ensindexstr}/da/restart.${timestr}.nc .
   do_restart='true'
 fi
-offset=$((10#${cyc}%6))
-CDATElbc=$($NDATE -${offset} ${CDATE})
+
+
+#offset=$((10#${cyc}%6))
+#CDATElbc=$($NDATE -${offset} ${CDATE})
+CDATElbc=${CDATE}
 ${cpreq} ${COMINrrfs}/${RUN}.${CDATElbc:0:8}/${CDATElbc:8:2}${ensindexstr}/lbc/lbc*.nc .
-${cpreq} ${FIXrrfs}/physics/${PHYSICS_SUITE}/* .
-ln -snf VEGPARM.TBL.fcst VEGPARM.TBL #gge.debug temp
+
+ln -snf ${FIXrrfs}/physics/${PHYSICS_SUITE}/* .
+ln -snf ${FIXrrfs}/meshes/3km_conus.ugwp_oro_data.nc .
 mkdir -p graphinfo stream_list
-${cpreq} ${FIXrrfs}/graphinfo/* graphinfo/
-cpreq ${FIXrrfs}/stream_list/${PHYSICS_SUITE}/* stream_list/
+ln -snf ${FIXrrfs}/graphinfo/* graphinfo/
+ln -snf ${FIXrrfs}/stream_list/${PHYSICS_SUITE}/* stream_list/
```

Comment:
For future references and discussion about where to use "ln -snf" and where to use "${cpreq}".
Per the NCO standard, we need to cpreq for any files which cannot be missed for running a task. But it looks like "ln -snf" also works. @MatthewPyle-NOAA @BenjaminBlake-NOAA Do you have any thoughts on what to use? Thanks!

Reply:
@guoqing-noaa I think a mix of both "cpreq" and "ln -snf" is okay to have.  If it's a large file then "ln -snf" would be preferred since copying over large files takes longer.

Reply:
@BenjaminBlake-NOAA Thanks for the input!